lib/buffer.js

```diff
@@ -270,6 +270,11 @@ function fromObject(obj) {
       }
       return fromArrayLike(obj);
     }
+    if (typeof SharedArrayBuffer !== 'undefined') {
```

Comment:
I removed the line on `fromObject()`. However I am unable to rebase interactively. For some reason git is unable to rebase.